lib/puppet/type/scheduled_task.rb

```diff
@@ -174,11 +174,11 @@ def insync?(current)
     end
 
     def should_to_s(new_value=@should)
-      self.class.format_value_for_display(new_value)
+      super(new_value)
     end
 
     def is_to_s(current_value=@is)
-      self.class.format_value_for_display(current_value)
+      super(current_value)
```

Comment:
I kept them to retain the default parameter assignment from `@is`. The super method doesn't do that.